Lib/test/test_typing.py

```diff
@@ -1482,6 +1482,15 @@ def test_cannot_instantiate(self):
         with self.assertRaises(TypeError):
             type(u)()
 
+    def test_cannot_iterate(self):
```

Comment:
Add test for `isinstance(Union, collections.abc.Iterable)`.

Reply:
Done.